server/config/config.go

```diff
@@ -118,14 +118,26 @@ func (c Config) MaxWaitTimeForTraceDuration() time.Duration {
 func (c Config) DataStore() (*TracingBackendDataStoreConfig, error) {
 	selectedStore := c.Server.Telemetry.DataStore
 	dataStoreConfig, found := c.Telemetry.DataStores[selectedStore]
-	if !found {
+
+	if selectedStore != "" && !found {
 		availableOptions := mapKeys(c.Telemetry.DataStores)
 		return nil, fmt.Errorf(`invalid data store option: "%s". Available options: %v`, selectedStore, availableOptions)
 	}
 
+	if !found {
+		return nil, nil
+	}
+
 	return &dataStoreConfig, nil
 }
 
+func (c Config) IsDataStoreConfigured() bool {
+	selectedStore := c.Server.Telemetry.DataStore
+	_, found := c.Telemetry.DataStores[selectedStore]
+
+	return found
```

Comment:
We could avoid logic duplication by reusing the `DataStore` method

```suggestion
	ds, err := c.DataStore()
	return err == nil && ds != nil
```